src/views/Dashboard/Lab/Account/index.vue

```diff
@@ -52,7 +52,7 @@
                   placeholder="Lab Name"
                   autocomplete="off"
                   outlined
-                  :disabled="!isEditable"
+                  :disabled="!isEditable || !isVerify"
```

Comment:
The issue only restricts the changing of Country, State, City, and Address. Not lab names.

Reply:
Is there a mistake in the issue? Please confirm with @hilyds on this issue.

Reply:
from the last time discussion with @hilyds , if lab status is verified, lab only can change email, phone number, website address and profile picture.